hack/update_catalog.sh

```diff
@@ -48,7 +48,7 @@ verify_bundle_image()
 	fi
 
 	log_info "inspecting ${OPERATOR_BUNDLE_IMAGE} bundle image"
-	media_type="$(podman run -e REGISTRY_AUTH_FILE="/tmp/auth.json" --rm -v "${auth_file}:/tmp/auth.json" \
+	media_type="$(podman run -e REGISTRY_AUTH_FILE="/tmp/auth.json" --rm -v "${auth_file}:/tmp/auth.json:z" \
```

Comment:
nit: no harm adding the shared label. Was access issue encountered? Did the host machine have any labels set on the auth file?